.travis.yml

```diff
@@ -11,7 +11,9 @@ install:
     - pip install codecov
     - export PYTHONPATH=.
 
-script: ./scripts/test --cov
+script:
+  - ./scripts/lint --flake8
```

Comment:
Ideally Travis would run the Linting as a seperate test run. (Eg see how the Travis builds against django rest framework look)